static/app/utils/replays/replayDataUtils.tsx

```diff
@@ -179,10 +179,13 @@ export function breadcrumbFactory(
     })
     .map(crumb => {
       if (crumb.category === 'replay.mutations') {
+        const crumbData = crumb.data as Record<string, unknown>;
         return {
           ...crumb,
-          type: BreadcrumbType.WARNING,
-          level: BreadcrumbLevelType.WARNING,
+          type: crumbData.limit ? BreadcrumbType.ERROR : BreadcrumbType.WARNING,
+          level: crumbData.limit
+            ? BreadcrumbLevelType.FATAL
```

Comment:
I wonder if level `FATAL` is the way to go here. On mobile/desktop/server it means the process crashed usually. How about:
```suggestion
            ? BreadcrumbLevelType.ERROR
```

Reply:
tbh not sure what level even does here